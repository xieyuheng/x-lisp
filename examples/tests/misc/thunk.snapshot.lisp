(lambda () #t)
((lambda () #t))

(define (f) #t)
f
(f)
