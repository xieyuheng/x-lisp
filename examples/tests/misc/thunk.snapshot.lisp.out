(lambda () #t)
#t
(lambda () #t)
#t
